### PR TITLE
IBM Power updates for enabling kdump

### PR DESCRIPTION
**Pull request by alishaIBM**
Version(s): 4.14 and previous versions

Issue: https://issues.redhat.com/browse/OCPBUGS-20031

Link to docs preview: https://65649--docspreview.netlify.app/openshift-enterprise/latest/support/troubleshooting/troubleshooting-operating-system-issues#enabling-kdump

QE review:
- [x] QE has approved this change.

Additional information:
Reference link: https://access.redhat.com/solutions/5907911

### CODE REVIEW

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: modules/troubleshooting-enabling-kdump.adoc [hunk 1]
```diff
@@ -18,6 +18,11 @@ Perform the following steps to enable kdump on {op-system}.
 ----
 # rpm-ostree kargs --append='crashkernel=256M'
 ----
++
+[NOTE]
+====
+For `ppc64le` platform the recommended value for `crashkernel` is `crashkernel=2G-4G:384M,4G-16G:512M,16G-64G:1G,64G-128G:2G,128G-:4G`. See the link:https://access.redhat.com/solutions/5907911[knowledgebase solution] for further information on how has the `crashkernel=auto` parameter changed between RHEL8 minor releases.
```
[the remark this answers is not in the document]
Comment (alishaIBM) [reply modules/troubleshooting-enabling-kdump.adoc]: Okay. Removed the additional Sentence.

**Comment by alishaIBM** [comment]
@mkumatag shall I go ahead with the RH peer review ?

File: modules/troubleshooting-enabling-kdump-day-one.adoc [hunk 2]
```diff
@@ -66,6 +66,7 @@ systemd:
 <2> Provide kernel arguments to reserve memory for the crash kernel. You can add other kernel arguments if necessary.
```
Comment (mkumatag) [inline modules/troubleshooting-enabling-kdump-day-one.adoc:66]: wondering if we can make the `crashkernel` suggestion here as well for power

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: modules/troubleshooting-enabling-kdump-day-one.adoc (hunk 2 above)
Comment (alishaIBM) [reply modules/troubleshooting-enabling-kdump-day-one.adoc]: Agree. Added the crashkernel suggestion for IBM Power.

fyi: @mkumatag

**Review by mkumatag — APPROVED** [review]
/lgtm

File: modules/troubleshooting-enabling-kdump-day-one.adoc [hunk 3]
```diff
@@ -63,9 +63,10 @@ systemd:
 ----
 +
 <1> Replace `worker` with `master` in both locations when creating a `MachineConfig` object for control plane nodes.
-<2> Provide kernel arguments to reserve memory for the crash kernel. You can add other kernel arguments if necessary.
+<2> Provide kernel arguments to reserve memory for the crash kernel. You can add other kernel arguments if necessary. For `ppc64le` platform the recommended value for `crashkernel` is `crashkernel=2G-4G:384M,4G-16G:512M,16G-64G:1G,64G-128G:2G,128G-:4G`.
```
Comment (mburke5678) [inline modules/troubleshooting-enabling-kdump-day-one.adoc:66]:
```suggestion
<2> Provide kernel arguments to reserve memory for the crash kernel. You can add other kernel arguments if necessary. For the `ppc64le` platform, the recommended value for `crashkernel` is `crashkernel=2G-4G:384M,4G-16G:512M,16G-64G:1G,64G-128G:2G,128G-:4G`.
```

File: modules/troubleshooting-enabling-kdump.adoc [hunk 4]
```diff
@@ -18,6 +18,11 @@ Perform the following steps to enable kdump on {op-system}.
 ----
 # rpm-ostree kargs --append='crashkernel=256M'
 ----
++
+[NOTE]
+====
+For `ppc64le` platform the recommended value for `crashkernel` is `crashkernel=2G-4G:384M,4G-16G:512M,16G-64G:1G,64G-128G:2G,128G-:4G`.
```
Comment (mburke5678) [inline modules/troubleshooting-enabling-kdump.adoc:24]:
```suggestion
For the `ppc64le` platform, the recommended value for `crashkernel` is `crashkernel=2G-4G:384M,4G-16G:512M,16G-64G:1G,64G-128G:2G,128G-:4G`.
```

**Comment by alishaIBM** [comment]
@mburke5678 thank you for the suggestions. I have updated the PR. Request you to check. Thanks.

**Pull request merged**